server/container/guice/guice-common/src/main/java/org/apache/james/utils/GuiceServerProbe.java

```diff
@@ -60,13 +59,13 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(GuiceServerProbe.class);
 
     private final MailboxManager mailboxManager;
-    private final MailboxMapperFactory<CassandraId> mailboxMapperFactory;
+    private final MailboxMapperFactory mailboxMapperFactory;
     private final DomainList domainList;
     private final UsersRepository usersRepository;
     private final SieveRepository sieveRepository;
 
     @Inject
-    private GuiceServerProbe(MailboxManager mailboxManager, MailboxMapperFactory<CassandraId> mailboxMapperFactory,
+    private GuiceServerProbe(MailboxManager mailboxManager, MailboxMapperFactory mailboxMapperFactory,
```

Comment:
wow, you probably introduced a lot of warning by removing generics. You should actually make thinks more generic instead of less generic